openmc/statepoint.py

```diff
@@ -437,10 +461,13 @@ def _read_tallies(self):
             # Add Tally to the global dictionary of all Tallies
             self.tallies[tally_key] = tally
 
-
     def read_results(self):
+        """Read tally results and store them in the ``tallies`` attribute. By default,
+        when the statepoint is first loaded, no results are read.
```

Comment:
Would read more clearly as "No results are read when the statepoint is instantiated."